R/pkg/vignettes/sparkr-vignettes.Rmd

```diff
@@ -776,6 +778,20 @@ newDF <- createDataFrame(data.frame(x = c(1.5, 3.2)))
 head(predict(isoregModel, newDF))
 ```
 
+#### Decision Tree
+
+`spark.decisionTree` fits a [decision tree](https://en.wikipedia.org/wiki/Decision_tree_learning) classification or regression model on a `SparkDataFrame`.
+Users can call `summary` to get a summary of the fitted model, `predict` to make predictions, and `write.ml`/`read.ml` to save/load fitted models.
+
+We use the `longley` dataset to train a decision tree and make predictions:
+
+```{r}
+df <- createDataFrame(longley)
```

Comment:
yes - but as mentioned, if you can think of a data set that doesn't have dot in column name, like `as.data.frame(Titanic)`